teuthology/task/install/rpm.py

```diff
@@ -62,6 +64,35 @@ def _remove(ctx, config, remote, rpm):
     else:
         remote.run(args='sudo yum clean expire-cache')
 
+def _yum_addrepo(remote, repo_list):
+    """
+    Add yum repos to the remote system.
+
+    :param remote: remote node where to add packages
+    :param repo_list: list of dictionaries with keys 'url'.
+    :return:
+    """
+    for repo in repo_list:
+        # add and enable the repo
+        remote.sh(f'sudo dnf config-manager --add-repo --refresh --no-gpgcheck {repo["url"]}')
+
+def _yum_removerepo(remote, repo_list):
+    """
+    Remove yum repos on the remote system.
+
+    :param remote: remote node where to remove packages from
+    :param repo_list: list of dictionaries with keys 'url'
+    :return:
+    """
+    for repo in repo_list:
+        # remove prefixed "https://" or "http://"
+        repo_name = repo['url'].replace("https://", "")
+        repo_name = repo_name.replace("http://", "")
+
+        # then replace all '/' with '_'
+        repo_name = repo_name.replace('/', '_')
```

Comment:
(style) the code is error prone, it would be much better for readability to use chaining and right to variable once:
```
repo_name = repo['url'].replace("https://", "") \
                       .replace("http://", "") \
                       .replace('/', '_')

```